[doc] Improve Pulsar Administration Pulsar dashboard (#5221)
apache/pulsar: site2/docs/administration-dashboard.md

The dashboard is a data collector that polls stats from all the brokers in a Pulsar instance (across multiple clusters) and stores all the information in a [PostgreSQL](https://www.postgresql.org/) database.

> You can use a [Django](https://www.djangoproject.com) web app to render the collected data.

Reviewer: ```suggestion
You can use the [Django](https://www.djangoproject.com) web APP to render the collected data.
```

  apachepulsar/pulsar-dashboard:{{pulsar:version}}
```

> You can find the {@inject: github:`Dockerfile`:/dashboard/Dockerfile} in `dashboard` directory, you can build a image from scratch also:

Reviewer: ```suggestion
You can find the {@inject: github:`Dockerfile`:/dashboard/Dockerfile} in the `dashboard` directory and build an image from scratch as well:
```
